OpenRA.Mods.Common/Traits/Render/SelectionDecorations.cs

```diff
@@ -119,7 +124,7 @@ IEnumerable<IRenderable> DrawPips(WorldRenderer wr, Actor self, int2 basePositio
 			if (!pipSources.Any())
 				yield break;
 
-			var pipImages = new Animation(self.World, "pips");
+			var pipImages = new Animation(self.World, Info.Image);
```

Comment:
and same for [line 133](https://github.com/Mailaender/OpenRA/blob/3a7558c273bdebd2e7466173331f1100d9ece1e8/OpenRA.Mods.Common/Traits/Render/SelectionDecorations.cs#L133)